eva-persistence-jdbc/src/main/kotlin/com/razz/eva/persistence/jdbc/executor/JdbcQueryExecutor.kt

```diff
@@ -17,21 +20,21 @@ import org.jooq.Table
 import org.jooq.exception.DataAccessException
 import org.jooq.impl.DSL
 import org.postgresql.util.PSQLException
-import java.sql.Connection
 
 class JdbcQueryExecutor(
     private val transactionManager: TransactionManager<Connection>,
-    private val tracer: Tracer = NoopTracerFactory.create()
+    private val openTelemetry: OpenTelemetry = noop(),
 ) : QueryExecutor {
 
     override suspend fun <R : Record> executeSelect(
         dslContext: DSLContext,
         jooqQuery: Select<R>,
         table: Table<R>,
+        tag: String?,
```

Comment:
I'd refrain from changing this interface. Can't we pass telemetry stuff in coroutine context?